File: cloud/dao/sys_logger_login.go
package dao

import "github.com/abulo/ratel/v3/stores/null"

// SysLoggerLogin 登录日志 sys_logger_login
type SysLoggerLogin struct {
	Id         *int64        `gorm:"primaryKey;autoIncrement;column:id" json:"id"` //bigint 编号,PRI
	Name       null.String   `gorm:"column:name" json:"name"`                      //varchar 姓名
	Username   *string       `gorm:"column:username" json:"username"`              //varchar 用户名
	UserId     *int64        `gorm:"column:user_id" json:"userId"`                 //bigint 用户编号
	Ua         null.String   `gorm:"column:ua" json:"ua"`                          //varchar UA
	LoginTime  null.DateTime `gorm:"column:login_time" json:"loginTime"`           //datetime 登录时间
	Channel    null.String   `gorm:"column:channel" json:"channel"`                //varchar 渠道
	Ip         null.String   `gorm:"column:ip" json:"ip"`                          //varchar IP
	Deleted    *int32        `gorm:"column:deleted" json:"deleted"`                //tinyint 删除:0否/1是
	TenantId   *int64        `gorm:"column:tenant_id" json:"tenantId"`             //bigint 租户
	Creator    null.String   `gorm:"column:creator" json:"creator"`                //varchar 创建人
	CreateTime null.DateTime `gorm:"column:create_time" json:"createTime"`         //datetime 创建时间
	Updater    null.String   `gorm:"column:updater" json:"updater"`                //varchar 更新人
	UpdateTime null.DateTime `gorm:"column:update_time" json:"updateTime"`         //datetime 更新时间
}

func (SysLoggerLogin) TableName() string {
	return "sys_logger_login"
}

File: admin/src/api/interface/sysLoggerLogin.ts
// sys_logger_login 登录日志
export interface ReqSysLoggerLoginParams extends RequestPage {
  deleted?: number // 删除:0否/1是
  channel?: string // 渠道
  beginLoginTime?: string // 登录时间
  finishLoginTime?: string
  username?: string // 用户名
  deptId?: number // 部门
}

export interface ResSysLoggerLogin {
  id: number // 编号
  name: string | undefined // 姓名
  username: string // 用户名
  userId: number // 用户编号
  ua: string | undefined // UA
  loginTime: string // 登录时间
  channel: string | undefined // 渠道
  ip: string | undefined // IP
  deleted: number // 删除:0否/1是
  tenantId: number // 租户
  creator: string | undefined // 创建人
  createTime: string | undefined // 创建时间
  updater: string | undefined // 更新人
  updateTime: string | undefined // 更新时间
}
